js/templates/ratingStars.html

```diff
@@ -1,15 +1,13 @@
 <% for (var i=0; i < ob.rating; i++) { %>
-<div class="inlineBlock positionRelative">
   <% if (Math.floor(ob.rating) === i) { %>
-    <span class="ion-android-star unfilled"></span>
-    <span class="ion-android-star textOpacity1 positionAbsolute noOverflow" style="top:0;bottom:0;left:0;right:0;width:<%= ((ob.rating % 1) * 100) + '%'%>"></span>
+<div class="inlineBlock positionRelative">
+  <span class="ion-android-star unfilled"></span>
+  <span class="ion-android-star textOpacity1 positionAbsolute noOverflow" style="top:0;bottom:0;left:0;right:0;width:<%= ((ob.rating % 1) * 100) + '%'%>"></span>
+</div>
   <% } else { %>
-    <span class="ion-android-star textOpacity1"></span>
+<span class="ion-android-star textOpacity1"></span>
   <% } %>
-</div>
 <% } %>
 <% for (var i=0; i < 5 - ob.rating; i++) { %>
-<div class="inlineBlock positionRelative">
-  <span class="ion-android-star unfilled"></span>
-</div>
```

Comment:
With the changes made to this file, the partial stars don't work. It would be best to revert to how it was before.